### DISCUSSION TITLE
About Field of Study

### DISCUSSION THREAD

**Discussion by CKtrace** [community/community#20784] [Discussions]
What field are you currently learning or interested in?

**Comment by melvincwng** [comment]
Hello @CKtrace, welcome to GitHub Community 👋

I'm interested in learning both front end and back end development 😊

Hope to connect with you on GitHub and collaborate on some projects in the future!

감사합니다내친구!

**Reply by CKtrace** [reply]
Thank you so much for your reply!  
The field you are interested in looks very good!
I have no doubt that @melvincwng will be more growth in interested field!!
Have a Nice Day!

**Comment by The-Real-Thisas** [comment]
Hi @melvincwng, Welcome 👋

I'm currently trying to figure out machine vision, specifically real time scene reconstruction via real time depth estimation. 

Cool website ! Also looking forward to potentially working with you.

Cheers,
Thisas

**Reply by melvincwng** [reply]
Hey @The-Real-Thisas , welcome man to GitHub Community 👋🏻. Thanks for your kind words as well ^^

Wow, machine vision! That's really impressive! Machine Learning is something that I hope to learn in the future, but as of now, I'm just trying to solidify my fundamental software engineering skills first.

Kudos to you, and I look forward to collaborating with you too in the future.

If you are interested, I have a very beginner-friendly open source repository/organization. You can take a look and contribute if you wish to, over here - https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project

It's main purpose is to allow new coders and introduce them the idea of contributing to open source projects :)

All the best in your studies for machine vision 👍🏻
